commit d26a72d8f491dbd90ad740437ec585d332994206
Author: MaxHearnden <maxoscarhearnden@gmail.com>
Date:   Fri Sep 11 02:59:43 2026 +0100

    ssh_filter_btrbk: Allow command options to be after paths

diff --git a/ssh_filter_btrbk.sh b/ssh_filter_btrbk.sh
index d8adcb0..964e961 100755
--- a/ssh_filter_btrbk.sh
+++ b/ssh_filter_btrbk.sh
@@ -94,7 +94,7 @@ reject_filtered_cmd()
     fi
 
     # allow multiple paths (e.g. "btrfs subvolume snapshot <src> <dst>")
-    allow_cmd_match="(${allow_list})( ${option_match})*( ${path_match})+"
+    allow_cmd_match="(${allow_list})( ${option_match})*( ${path_match})( (${option_match}|${path_match}))*"
     stream_in_match="(${decompress_match} \| )?(${mbuffer_match} \| )?"
     stream_out_match="( \| ${mbuffer_match})?( \| ${compress_match})?"
 
